modules/virt-configuring-vm-project-dpdk.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/vm_networking/virt-attaching-vm-to-sriov-network.adoc
+
+:_content-type: PROCEDURE
+[id="virt-configuring-vm-project-dpdk_{context}"]
+= Configuring a project for DPDK workloads
+
+Configure the project to run DPDK workloads on SR-IOV hardware. 
+
+.Prerequisites
+* Your cluster is configured to run DPDK workloads.
+
+.Procedure
+. Create a namespace for your DPDK applications:
++
+[source,terminal]
+----
+$ oc create ns dpdk-checkup-ns
+----
```

Comment:
@phoracek Is there a difference between creating a namespace or creating a project?

Reply:
Frankly, I don't know if OpenShift handles the two differently. I would stick to whatever we use elsewhere in the docs

Reply:
@orelmisan This is the documentation on projects that I have linked to in the Additional resources section: https://docs.openshift.com/container-platform/4.12/applications/projects/working-with-projects.html

Is it relevant for this procedure?

Reply:
> @orelmisan This is the documentation on projects that I have linked to in the Additional resources section: https://docs.openshift.com/container-platform/4.12/applications/projects/working-with-projects.html
> 
> Is it relevant for this procedure?

I think so, thanks.